config.schema.yaml

```diff
@@ -3,6 +3,11 @@ isp_db:
   type: "string"
   required: true
   default: "/opt/geoip2/GeoIP2-ISP.mmdb"
+asn_db:
```

Comment:
Would be good to also clarify those different database settings in the README:

* Are those different multiple databases mutually exclusive?
* Can they be used together? What happens if they are used to gether (all the information is included in the result otherwise just partial or similar)?
* etc